crates/driver/src/boundary/settlement.rs

```diff
@@ -213,6 +214,7 @@ impl Settlement {
             gas_price,
             &gas.into(),
         );
+        ensure!(!inputs.objective_value().is_negative(), "negative score",);
```

Comment:
```suggestion
        ensure!(!inputs.objective_value().is_negative(), "negative score");
```

`rustfmt` doesn't remove trailing commas in macro invocations 🤷.

Reply:
https://github.com/cowprotocol/services/pull/1526/commits/d70391d60e725367c3b95905ceaa0344f51e0040